plugins/graphql-kotlin-maven-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/maven/GenerateGraalVmMetadataMojo.kt

```diff
@@ -34,7 +34,7 @@ class GenerateGraalVmMetadataMojo : AbstractSourceMojo() {
     override lateinit var outputDirectory: File
 
     override fun generate() {
-        val metadataDirectory = File(outputDirectory, "META-INF/native-image/${project.groupId}/${project.name}")
+        val metadataDirectory = File(outputDirectory, "META-INF/native-image/${project.groupId}/${project.name}/graphql")
```

Comment:
GraalVM plugin looks for reachability metadata under any `META-INF/native-image` subdirectories. Per GraalVM documentation they suggest to generate reachability metadata under `<groupId>/<artifactId>` subdirectory to avoid collisions. Adding explicit `graphql` subdirectory to avoid collision with SpringBoot plugin generating metadata under the same directory.